aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move

```diff
@@ -35,10 +34,16 @@ module aptos_framework::aggregator {
     }
 
     /// Adds `value` to aggregator. Aborts on overflowing the limit.
-    public native fun add(aggregator: &mut Aggregator, value: u128);
+    public native fun add(
```

Comment:
Is this necessary? Feel like files will end up being double the code lines due to this.

Reply:
Here, the line break is due to the length of the parameters inside the parentheses being too long.

Reply:
fixed at https://github.com/movebit/movefmt/commit/9ca658736a254a5696676f95c8da7b498a90281c